deps/build.jl

```diff
@@ -2,14 +2,15 @@ using BinDeps
 using Compat
 
 vers = "1.8.1"
+const OSNAME = is_windows() ? :Windows : Compat.KERNEL
 
 tagfile = "installed_vers"
 target = "libblosc.$(Libdl.dlext)"
 url = "https://bintray.com/artifact/download/julialang/generic/"
 if !isfile(target) || !isfile(tagfile) || readchomp(tagfile) != "$vers $(Sys.WORD_SIZE)"
-    if OS_NAME == :Windows
+    if OSNAME == :Windows
```

Comment:
I'd rather just use `Compat.KERNEL == :NT`.   That seems cleaner than keeping the obsolete `:Windows` symbol around in perpetuity.